docs/install/Knative-with-MicroK8s.md

```diff
@@ -30,9 +30,7 @@ sudo snap alias microk8s.kubectl kubectl
 ```shell
 echo 'N;' | microk8s.enable knative
 ```
-This command will install Knative Serving version 0.7.1 and Eventing version 0.7.1 components.
```

Comment:
This might need to be validated. Its not clear to me if MicroK8s does this, but I do know some packages can remain on back versions.

Reply:
Managed to find this conversation: https://github.com/knative/docs/pull/1747#discussion_r325378124
I've reached out to @ammarn911

Reply:
Should we move this into a separate PR to avoid blocking all the other changes here?

Reply:
Yeah sounds good to me :slightly_smiling_face: do you want to raise a minor issue for this so we remember? :smile:

Reply:
That would be great, and then you can assign it to @ammarn911. Thanks @abrennan89 !

Reply:
@abrennan89 the current versions are 0.9.0

Reply:
Thanks @ammarn911 ! @RichieEscarez I'll update this to 0.9.0 then and we can merge?